src/main.py

```diff
@@ -1,20 +1,67 @@
+import os
+import toml
+import argparse
+
 from generate_csv import generate_csv
 from csv_to_cda import csv_to_cda
 from calculate_dependencies import calculate_dependencies
-from src.sent_to_server_and_print_error import send_xml_file
 
-if __name__ == '__main__':
-    csv_path = '../resources/data.csv'
-    excel_path = '../resources/CDAVariables.xlsx'
-    xslt_file = '../resources/EmergencyNote.xslt'
 
-    # First step: Generate
-    generate_csv(excel_path, csv_path, 10)
+def set_environment_variables(config):
+    """
+    Set environment variables based on the provided configuration dictionary.
+
+    Args:
+        config (dict): Configuration dictionary containing paths for environment variables.
+
+    Returns:
+        None
+    """
+    env_vars = [
+        ('CSV_PATH', 'cda_paths.csv_path'),
+        ('EXCEL_PATH', 'cda_paths.excel_path'),
+        ('XSLT_FILE', 'cda_paths.xslt_file'),
+        ('OUTPUT_DIR', 'cda_paths.output_dir'),
+        ('CITIES_CSV', 'csv_paths.cities_csv'),
+        ('DIAGNOSES_CSV', 'csv_paths.diagnoses_csv')
+    ]
+
+    for env_var, config_key in env_vars:
+        section, key = config_key.split('.')
+        os.environ[env_var] = config[section][key]
+
+
+def parse_command_line():
+    global parser, args, n
+    parser = argparse.ArgumentParser(description='Process Excel to CDA.')
+    parser.add_argument('--n', type=int, required=True, help='Number of patients to generate.')
+    args = parser.parse_args()
+    n = args.n
+
+
+def main():
+    config = toml.load('../config.toml')
```

Comment:
There should be no relative paths for using this script... Provide the config.toml also as an input argument. If I run

```shell
python3 /home/user/PycharmProjects/cda-test-data-generator/src/main.py
```

or

```shell
cd /home/user/PycharmProjects/cda-test-data-generator/src/
python3 main.py
```

both should work